catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -94,50 +79,61 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
     [images]
   );
 
-  const imageRenderer = useCallback(galleryImage => {
-    const photo: GalleryImageProps = galleryImage.photo;
+  const imageRenderer: React.FC<RenderPhotoProps<GalleryImageProps>> = ({
+    photo,
+    layout,
+    wrapperStyle,
```

Comment:
are these used/defined? Are they from the library?

Reply:
clarifying comment added, they are passed in by <PhotoAlbum/>